protocol/signalfx/format/signalfx_format.go

```diff
@@ -98,3 +98,19 @@ type InputSpan struct {
 // InputSpanList is an array of InputSpan pointers
 //easyjson:json
 type InputSpanList []*InputSpan
+
+// JSONLogHECV1 decodes enterprise HEC format, details: https://docs.splunk.com/Documentation/SplunkCloud/latest/Data/FormateventsforHTTPEventCollector#Event_metadata
+//easyjson:json
+type JSONLogHECV1 struct {
+	Time       *int64                 `json:"time,omitempty"`
```

Comment:
Do we need to make some fields not have omitempty?

Reply:
based on Document 

> The keys in Event Metadata are all optional.